atd-etl/cris_import/cris_import.py

```diff
@@ -612,7 +612,23 @@ def align_records(map_state):
             input_column_names = util.get_input_column_names(pg, map_state["import_schema"], table, target_columns)
 
             # iterate over each imported record and determine correct action
+            should_skip_update = False
+
             for source in imported_records:
+                # Check unique key columns to make sure they all have a value
+                for key_column in key_columns:
+                    key_column_value = source[key_column]
+                    if key_column_value == None:
```

Comment:
super minor, but i always see this test written as `is None` instead of `== None`. there is some arcane pythonic reason for it that i am too lazy to look up 😅

Reply:
sounds great 🐍 I'll push this up along with the other minor ones above. Thanks! 🙏